src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -416,6 +410,8 @@ private void purgeUnusedRefDBs()
 	@SuppressWarnings("unchecked")
 	private void createReferences(long personID, Map<String, Map<String, ?>> dbMappings) throws IOException, Exception
 	{
+		String linkCheckReportName = LINK_CHECK_REPORTS_PATH + "/linkCheckSummaryReport" + DateTimeFormatter.ofPattern(DATE_PATTERN_FOR_FILENAMES).format(LocalDateTime.now()) + ".tsv";
+		Files.write(Paths.get(linkCheckReportName), "RefDBName\tNumOK\tNumNotOK\n".getBytes(), StandardOpenOption.CREATE);
```

Comment:
Should the link check report have the URL in it? Or if that is adding too much text should we have the URL template somewhere?

Reply:
This report is a summary of counts for each resource: how many OK, how many not OK.

More detailed info about, possiblty for each URL, should be in the LinkChecker log or LinkCheckManager log.